### PR TITLE
Lot of fixes, improvements etc...

### PR DESCRIPTION
**Pull request by QuentinN42**
Here is a list of some improvements made to clairvoyance:

- fixed a lot of regexes
- `0c4902e` : feat: progress bar as an optional parameter
- `099fba1` : feat: use pathlib to work with any OS
- `cdf0076` : docs: update readme
- `df6b6d2` : fix: don't crash when no field suggestion
- `b5e8c71` : feat: disable ssl verification as an optional parameter
- `d7e7e63` : fix: reccursive schemas loops
- `f842156` : ignore aiohttp warns
- `ac00768` : @jamboro fixes issue 11
- `36a0217` : replace prints by logger
- `f5fa684` : feat: add backoff as an optional parameter
- `038c0f4` : feat: add max retry param
- `2e97088` : feat: proxy as an optional parameter

### CODE REVIEW

**Review by QuentinN42 — COMMENTED** [review]
Ok

**Review by QuentinN42 — COMMENTED** [review]
First review done, need to test it on any python versions.

**Comment by QuentinN42** [comment]
Working from python 3.8 to python 3.11

**Comment by nikitastupin** [comment]
Thanks @QuentinN42! I'm going to review as soon as possible.

File: pyproject.toml [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+version = "2.5.1"
 description = "Obtain GraphQL API Schema even if the introspection is not enabled"
 authors = [
     "Nikita Stupin <nikitastupin@protonmail.com>",
     "Swan <swan@escape.tech>"
 ]
 maintainers = [
     "Nikita Stupin <nikitastupin@protonmail.com>",
+    "Antoine Carossio <antoine@escape.tech>",
+    "Swan <swan@escape.tech",
```
Comment (nikitastupin) [inline pyproject.toml:12]: Closing angle bracket?

Comment (QuentinN42) [reply pyproject.toml]: Thanks !

File: README.md [hunk 2]
```diff
@@ -1,60 +1,48 @@
...
-docker run --rm -v $(pwd):/tmp/ nikitastupin/clairvoyance -vv -o /tmp/schema.json -w /tmp/wordlist.txt https://swapi-graphql.netlify.app/.netlify/functions/index
-```
-
-### From BlackArch Linux
-
-> NOTE: this distribution is supported by a third-party (i.e. not by the mainainters of clairvoyance)
-
-```bash
-pacman -S clairvoyance
-```
```
Comment (nikitastupin) [inline README.md:57]: Do you mind bringing this back? Any arguments against this?

**Review by nikitastupin — APPROVED** [review]
I'd like to introduce small edits but they're not related to code. So I'm going to merge this and edit in the main branch. Thank you very much for the contributions 💪 Looks amazing 👏

**Pull request merged**